readme.md

```diff
@@ -39,8 +40,10 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 Your Answer:
 ```text
 
-```
+SQL databases are primarily called as Relational Databases; whereas NoSQL database are primarily called as non-relational or distributed database.
```

Comment:
These names are appropriate, but I normally hear NoSQL referred to as document databases.